cli/ceph/orch/tuned_profile.py

```diff
@@ -0,0 +1,70 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class TunedProfile(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(TunedProfile, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} tuned-profile"
+
+    def apply(self, spec_file, **kw):
+        """
+        apply tuned profile.
+        Args:
+            instance: cluster instance
+            config: key/value pairs that needs to be provided to the installer
```

Comment:
If these parameter are mandatory then instead of using `kw` pass these variables as method signature

Reply:
Updated